client.lua

```diff
@@ -0,0 +1,20 @@
+local lunch = require 'luncheon'
```

Comment:
This file seems to be basically repeated in the `examples` directory. Do we need both?

Reply:
Good call, it really should be part of the .gitignore